fuse/ipns/common.go

```diff
@@ -20,23 +18,14 @@ func InitializeKeyspace(n *core.IpfsNode, key ci.PrivKey) error {
 		return err
 	}
 
-	ctx, cancel := context.WithCancel(n.Context())
-	defer cancel()
```

Comment:
this should probably stay in this func, but be moved all the way to the top, to cover `n.DAG.Add` as well as `pub.Publish`